erpnext/setup/setup_wizard_slide/welcome_to_erpnext!/welcome_to_erpnext!.json

```diff
@@ -7,13 +7,13 @@
  "domains": [],
  "help_links": [],
  "idx": 0,
- "image_src": "/assets/erpnext/images/illustrations/onboard.png",
+ "image_src": "/assets/erpnext/images/illustrations/desk-onboard.png",
```

Comment:
Can we make the illustrations a bit darker? They do blend into the product but too light to be seen. If we are allocating space for the illustrations, they can be a bit darker and noticeable.

Reply:
The color of the illustrations is deliberate for the following reasons.
1. The rest of the product has no illustrations what-so-ever, so anything bold will be inconsistent with the overall feel of the product
1. The extreme light nature makes them as minimal as possible. The intention is for it to not be noticeable, so that the user can instead focus on the information he's supposed to enter.
1. The choice of not using any other colors was deliberate, because the product uses color explicitly to convey status information.
1. The colors used in the product lie between the standard colors used in the product.

However there is a good case for having better colors and more contrast for a few reasons.
1. Higher contrast has a accessibility value to it.
1. The current colors is not entirely from the standard palette anyway.. Might as well go all out with bolder illustrations and make it a fresher approach

@netchampfaris @surajshetty3416 @ruchamahabal I could use some feedback on the points above...